leaderboardFeature/src/main/java/com/panhuk/leaderboardfeature/ui/LeaderboardViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.panhuk.leaderboardfeature.ui
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.panhuk.domain.model.Leaderboard
+import com.panhuk.leaderboardfeature.R
+import com.panhuk.repository.LeaderboardRepo
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class LeaderboardViewModel @Inject constructor(
+  private val leaderboardRepo: LeaderboardRepo,
+  private val coroutineDispatcher: CoroutineDispatcher
+) : ViewModel() {
+
+  private lateinit var leaderboard: List<Leaderboard>
```

Comment:
plural?

Reply:
I guess it's better to left it like this. Or change the entire data class record but I don't think that it will be a better approach

Reply:
i meant variable name
is smth is a list plural naming emphasizes it

Reply:
maybe just leaderboard -> leaderboards

Reply:
yes, exactly this